### DISCUSSION TITLE
Implement custom error pages

### DISCUSSION THREAD

**Discussion by ianfirme** [community/community#59689] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I implement custom error pages for my Github Pages website?

**Comment by tcortega** [comment]
To implement custom error pages, you simply need to create a 404.html in the root directory. The server will automatically use this file when an error occurs. One caveat is that GitHub Pages only supports static pages, so any server-side scripting like PHP will not work. Additionally, GitHub uses the Jekyll theme’s 404 page by default, not your custom one, if you’re using a theme that has a 404 page. Also, 404s will not work when testing your pages locally with Jekyll.

**Comment by queenofcorgis** [comment, marked as answer]
@ianfirme @tcortega GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
